spring-cloud-azure-samples/spring-cloud-azure-sql-sample/src/main/resources/schema.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE users (
+  email VARCHAR(255),
+  first_name VARCHAR(255),
+  last_name VARCHAR(255),
+  PRIMARY KEY (email));
```

Comment:
Not sure whether I understood correctly, if the user needs to start this sample up, at least the user has to create the table and insert data with the sql files provided here, so we are not considering initialize this for them?

Reply:
This sql will be automatically executed when starting up.